rom/src/rom.rs

```diff
@@ -236,7 +236,21 @@ impl Soc {
         self.registers
             .fuse_soc_stepping_id
             .write(soc::bits::FuseSocSteppingId::SocSteppingId.val(soc_stepping_id));
-        // TODO: debug unlock / rma token?
+
+        if fuses.cptra_ss_manuf_debug_unlock_token().len()
+            != self.registers.fuse_manuf_dbg_unlock_token.len() * 4
```

Comment:
nit: This looks fine, but I've seen comparisons like this go wrong in the past. Can we use `size_of_val` instead?


```suggestion
        if size_of_val(&fuses.cptra_ss_manuf_debug_unlock_token())
            != size_of_val(&self.registers.fuse_manuf_dbg_unlock_token)
```

Reply:
good point. fixed.